src/js/serverSetup.js

```diff
@@ -22,8 +22,8 @@ fluid.defaults("sjrk.storyTelling.server", {
                 globalConfig: {
                     binaryUploadDirectory: "./uploads",
                     uploadedFilesHandlerPath: "/uploads",
+                    deletedFilesRecoveryPath: "/deleted_uploads",
```

Comment:
Is there a JIRA created for adding tests for this new endpoint?

Reply:
I have filed a Jira to address this: https://issues.fluidproject.org/browse/SJRK-208

Please feel free to make any suggestions if it does not cover what you imagined!